packages/techdocs-common/src/stages/generate/index.ts

```diff
@@ -15,4 +15,8 @@
  */
 export { TechdocsGenerator } from './techdocs';
 export { Generators } from './generators';
-export type { GeneratorBuilder, GeneratorBase } from './types';
+export type {
+  GeneratorBuilder,
+  GeneratorBase,
+  GeneratorRunOptions,
```

Comment:
It wasn't exported before but I find it helpful if it is part of the `api-report.md`.